### ISSUE TITLE
About the ViconFemalexxxxxx.fbx file

### ISSUE THREAD

**Issue by Yixin-Shen-1218** [YoungSeng/DiffuseStyleGesture#36]
Hi,

I didn't find this file in the GitHub repo or the blender market, how can I get this file?

Thank you in advance!

**Comment by YoungSeng** [comment]
Yes, I didn't provide this file because of possible copyright issues. I think this file you can get on the official Vicon website or you can use the fpx in maximo for example `Ch09_nonPBR.fbx` [here](https://github.com/YoungSeng/DiffuseStyleGesture/blob/master/visualize_gesture_using_Blender.md).

**Comment by Yixin-Shen-1218** [comment]
Oh thank you. I have tested using the `Ch09_nonPBR.fb` in maximo.

But I still can not find the file on the official Vicon website. Would you mind providing the resource link to the official Vicon website?

Thank you in advance!

**Comment by YoungSeng** [comment]
Sorry, I haven't looked for this; I'm not sure if it's from the company and if it's completely risk-free (it may be part of the company's assets), so sharing it wouldn't be a good idea, and I thought you might be able to look for it again. I hope you understand me.
